Clarify playable branch link comment (#67)
endlessm/amalgamate-pages: comment.md

{% if url %}

> Play this branch (full build). Launches the game from the start, not directly where the change was made {{ url }}.

Reviewer: How about this:

```suggestion
Play this branch at {{ url }}.

(This launches the game from the start, not directly at the change(s) in this pull request.)
```
